src/en/clouds-maas.md

```diff
@@ -4,89 +4,114 @@ Title: Using MAAS with Juju
 
+Enter a name for your maas cloud: maas-cloud
 
-Enter a name for your maas cloud: mainmaas
-
-Enter the API endpoint url: http://maas.example.org:5240/MAAS
+Enter the API endpoint url: http://10.55.60.29:5240/MAAS
 
-Cloud "mainmaas" successfully added
-You may bootstrap with 'juju bootstrap mainmaas'
+Cloud "maas-cloud" successfully added
+You may bootstrap with 'juju bootstrap maas-cloud'
 ```
 
-This will add both the 'mainmaas' cloud, which you can confirm
-by running:
+We've called the new cloud 'maas-cloud' and used an endpoint of
+'http://10.55.60.29:5240/MAAS'.
+
+Now confirm the successful addition of the cloud:
 
 ```bash
 juju clouds
 ```
 
-This will list the newly added clouds, preceded with the prefix 'local:' which
-denotes that these are local clouds added by the user:
+Here is a partial output:
 
 ```no-highlight
-Cloud        Regions  Default        Type        Description
-aws               11  us-east-1      ec2         Amazon Web Services
-...
-mainmaas            0                 maas        Metal As A Service
+Cloud        Regions  Default          Type        Description
+.
+.
+.
+maas-cloud         0                   maas        Metal As A Service
 ```
 
-Before you bootstrap this cloud, it is necessary to add the relevant
-credentials, as explained below.
+### Manually defining MAAS clouds
+
+Alternatively, it is possible to manually define a single or multiple MAAS
+clouds with a file and then add a cloud by referring to such a file (still with
+`juju add-cloud`).  See [Manually adding MAAS clouds][maas-manual] for details.
 
-## Adding your MAAS credentials
+## Add credentials
+
+Use the interactive `add-credential` command to add your credentials to the new
+cloud:
 
 ```bash
-juju add-credential mainmaas
+juju add-credential maas-cloud
+```
+
+Example user session:
+
+```no-highlight
+Enter credential name: maas-cloud-creds
+
+Using auth-type "oauth1".
+
+Enter maas-oauth:
+
+Credentials added for cloud maas-cloud.
 ```
 
-When prompted for "maas-oauth", you should paste your MAAS API key. Your API
-key can be found in the Account/User Preferences page in the MAAS web
-interface, or by using the MAAS CLI:
+We've called the new credential 'maas-cloud-creds' and used a MAAS user's API
+key in the place of 'maas-oauth'.
+
+!!! Note:
+    The API key will not be echoed back to the screen.
+
+Typically you will have a MAAS user of your own. The [MAAS API key][maas-api]
+can be found on your user preferences page in the MAAS web UI, or by using the
+[MAAS CLI][maas-cli], providing you have sudo access:
 
 ```bash
-sudo maas-region apikey --username=<user>
+sudo maas-region apikey --username=$PROFILE
 ```
 
-**Note:** Juju does not echo this key back to the screen.
+Where $PROFILE is to be replaced by the MAAS username.
+
+## Create the Juju controller
 
-Now you can create a Juju controller with the bootstrap command:
+You are now ready to create a Juju controller (this will utilize a node in the
```

Comment:
1. It is 'utilise'
2. I think it is probably better to explain it underneath (see next note)